src/Slides.js

```diff
@@ -1,21 +1,26 @@
 import { URL } from 'url';
 
+const getTrimmedPathName = str => str.replace(/\/+$/, '');
```

Comment:
Make this more clear by renaming `str` to `pathName`

```suggestion
const getTrimmedPathName = pathName => pathName.replace(/\/+$/, '');
```